### ISSUE TITLE
Esc-ing from a mod's description closes the entire management module.

### ISSUE THREAD

**Issue by Rensik** [gorilla-devs/GDLauncher#647]
**Describe the bug**

Esc-ing from a mod's description closes the entire management module.

**To Reproduce**

When viewing a mod's description, use the Esc key to back out of it.

**Expected behavior**

The module should back out to either the mod selection module or the mods included module, depending on where the mod that was selected was.

**Comment by RemagOfficial** [comment]
please fill in the whole report or remove unused parts

**Comment by Rensik** [comment]
Edited

**Comment by Eskaan** [comment]
I dont think we will be fixing this in the next time. The problem is, that we bound the `esc` key to the modals but not to the popup. It would require an extra state and an condition calling this state in the modals manager. Just to much efford involved to fix it.
